### DISCUSSION TITLE
Menu separator

### DISCUSSION THREAD

**Discussion by maxoriggi** [ttscoff/bunch-docs#28] [Ideas]
One quick feature request, which should be generally useful: a command to add a menu separator between bunches.
With the increasing number of bunches it would be nice to group them using separators.

**Comment by ttscoff** [comment]
I'm not sure how that would work, any thoughts? It would have to be part of a Bunch file as that's how the menu is built. Could have a frontmatter key that allowed you to put a separator before or after the item, maybe, or add a Bunch separator filetype. BUT, positioning a separator at a particular point would be finicky, I think.

**Reply by maxoriggi** [reply]
Yes, I was thinking about separator before/after keys in frontmatter. 
Clearly it makes sense only when you use menu order for sorting, in this case the separator will end up in a precise location in the menu. Without sorting it would be useless.

**Comment by ttscoff** [comment]
Next update adds frontmatter `menu divider` which can be `before` or `after`, or `true` to default to `after`. Actually pretty useful, now that I've played around with it for a while.

**Reply by maxoriggi** [reply]
Great, thanks!
